### PR TITLE
feat: added modal animation

### PR DESCRIPTION
**Pull request by impelcrypto**
**Pull Request Summary**

* Added modal animation
* Placed the modals at the top of the screen for mobile devices

**Check list**

- [ ] contains breaking changes
- [ ] adds new feature
- [x] modifies existing feature (bug fix or improvements)
- [ ] relies on other tasks
- [ ] documentation changes

Demo:
https://user-images.githubusercontent.com/92044428/162708496-259d2f62-66b4-44d6-be49-32af12c32b89.mov

image

### CODE REVIEW

**Comment by Kahonnohak** [comment]
Lovely! Can we confirm that mobile modals don't have the zooming animation? I checked on iphone, was great and checked on my computer dev tool, it zooms.

**Comment by impelcrypto** [comment]
@Kahonnohak 
Noted, I'll remove the zoom-in animation for mobile devices.

**Review by sirius651 — CHANGES_REQUESTED** [review]

File: package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
   },
   "dependencies": {
     "@astar-network/astar-api": "^0.1.5-3",
-    "@astar-network/astar-ui": "0.0.15",
+    "@astar-network/astar-ui": "0.0.17",
```
[the remark this answers is not in the document]
Comment (impelcrypto) [reply package.json]: @sirius651  Fixed

**Comment by impelcrypto** [comment]
@Kahonnohak 

> Can we confirm that mobile modals don't have the zooming animation? I checked on iphone, was great and checked on my computer dev tool, it zooms.

Fixed.

**Review by sirius651 — APPROVED** [review]

**Comment by Kahonnohak** [comment]
Thank you!

**Pull request merged**